plugins/convertColors.js

```diff
@@ -17,7 +18,8 @@ var collections = require('./_collections'),
     rNumber = '([+-]?(?:\\d*\\.\\d+|\\d+\\.?)%?)',
     rComma = '\\s*,\\s*',
     regRGB = new RegExp('^rgb\\(\\s*' + rNumber + rComma + rNumber + rComma + rNumber + '\\s*\\)$'),
-    regHEX = /^\#(([a-fA-F0-9])\2){3}$/;
+    regHEX = /^\#(([a-fA-F0-9])\2){3}$/,
+    none = /none/i;
```

Comment:
Probably, it's good idea to make it a bit more specific like at least `/\bnone\b/`.